src/main/java/me/leoko/advancedban/bungee/listener/ConnectionListenerBungee.java

```diff
@@ -1,6 +1,8 @@
 package me.leoko.advancedban.bungee.listener;
 
 import com.imaginarycode.minecraft.redisbungee.RedisBungee;
+import de.dytanic.cloudnet.driver.CloudNetDriver;
+import de.dytanic.cloudnet.ext.bridge.player.IPlayerManager;
```

Comment:
This class is not reserved for only CloudNet users and thus should not import any CloudNet API. This import looks unused anyway 🤔